src/ext/java/org/opentripplanner/ext/transmodelapi/model/plan/TripQuery.java

```diff
@@ -150,6 +151,16 @@ public static GraphQLFieldDefinition create(
           .type(new GraphQLNonNull(LocationInputType.INPUT_TYPE))
           .build()
       )
+      .argument(
+        GraphQLArgument
+          .newArgument()
+          .name("passThroughPoints")
+          .description(
+            "The points the journey is required to pass by, going from the start location to the destination."
```

Comment:
It's called passThroughPoint but the documentation talks about passing _by_. Is this intentional?

Reply:
Not really. I have fixed this and some other instances.